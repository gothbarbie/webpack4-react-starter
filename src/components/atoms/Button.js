import React from 'react'
import styled from 'styled-components'

/** @component */
export const Button = styled.button``
