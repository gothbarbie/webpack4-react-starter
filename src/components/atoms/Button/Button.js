import styled from 'styled-components'

const Button = styled.button``

/** @component */
export default Button
