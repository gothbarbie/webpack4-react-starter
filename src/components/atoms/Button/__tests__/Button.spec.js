import React from 'react'
import { shallow } from 'enzyme'

import Button from '../Button'

describe('components/atoms/Button', () => {
  it('renders', () => {
    const component = shallow(<Button>Test</Button>)

    expect(component).toMatchSnapshot()
  })
})
