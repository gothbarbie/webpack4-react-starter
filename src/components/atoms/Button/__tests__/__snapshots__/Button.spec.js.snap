// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/Button renders 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "Button-sc-12a0ki3-0",
        "isStatic": true,
        "rules": Array [],
      },
      "displayName": "Button",
      "render": [Function],
      "styledComponentId": "Button-sc-12a0ki3-0",
      "target": "button",
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Test
</StyledComponent>
`;
